packages/mobile-top-nav/src/media-menu.js

```diff
@@ -3,124 +3,194 @@ import { LitElement, html, css } from 'lit-element';
 import './media-slider';
 import './assets/img/media-menu-images';
 
+const menuSelection = [
+  {
+    icon: 'web',
+    label: 'Wayback Machine',
+  },
+  {
+    icon: 'texts',
+    label: 'Texts',
+  },
+  {
+    icon: 'video',
+    label: 'Video',
+  },
+  {
+    icon: 'audio',
+    label: 'Audio',
+  },
+  {
+    icon: 'software',
+    label: 'Software',
+  },
+  {
+    icon: 'images',
+    label: 'Images',
+  },
+  {
+    icon: 'more',
+    label: 'More',
+  },
+];
+
 class MediaMenu extends LitElement {
   static get properties() {
     return {
       mediaMenuOpen: { type: Boolean },
       mediaMenuAnimate: { type: Boolean },
       mediaSliderOpen: { type: Boolean },
-      mediaSliderAnimate: { type: Boolean }
+      mediaSliderAnimate: { type: Boolean },
+      selected: { type: String },
```

Comment:
Minor, but I might call this `selectedMenuOption` or something like that. `selected` made me think it might be a boolean (ie this this is selected or not)

Reply:
ha. i overthought that name last night